playbooks/roles/mongo_3_0/tasks/main.yml

```diff
@@ -322,7 +322,6 @@
 - name: create mongodb users in a replica set
   mongodb_user:
     database: "{{ item.database }}"
-    login_database: 'admin'
```

Comment:
@mrgnr It was throwing an unsupported arg error. this parameter is added in ansible 2.0 ( [check the docs](http://docs.ansible.com/ansible/mongodb_user_module.html) ) and we are using the `1.9.3-edx`.